File: БР1.1/Захматов Юрий/labs/lab1/src/main/java/com/petproject/itmoacsbackend/reviews/dto/ReviewResponse.java
package com.petproject.itmoacsbackend.reviews.dto;

import lombok.Builder;

import java.time.LocalDateTime;

@Builder
public record ReviewResponse(
        Long id,
        Long userId,
        Long propertyId,
        Integer rating,
        String comment,
        LocalDateTime createdAt
) {

}

File: БР1.2/Цатинян Артём/labs/lab1/src/main/kotlin/ru/itmo/restaurantbooking/review/adapter/rest/dto/ReviewResponse.kt
package ru.itmo.restaurantbooking.review.adapter.rest.dto

import java.time.LocalDateTime

data class ReviewResponse(
    val id: Long,
    val bookingId: Long,
    val userId: Long,
    val rating: Int,
    val comment: String?,
    val createdAt: LocalDateTime
)
